pyodbc.cpython-37m-x86_64-linux-gnu.so,sha256=rZbrywVYNnFHj_XojohALu5FLjN4ouYgJrk8kU0zAhQ,895209
pyodbc.pyi,sha256=rx05N8EYqujKI3FWtMikUjqMPTv4VQg0ChXny9DaisE,32326
pyodbc-5.0.0b3.dist-info/RECORD,,
pyodbc-5.0.0b3.dist-info/WHEEL,sha256=PptpQvQpMj4c6zoTgYAOmR3hDGeYN43ZmgdEwmXeaM8,150
pyodbc-5.0.0b3.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-5.0.0b3.dist-info/METADATA,sha256=2xC6FIGYQL7yD3CWWcBDeIY8vib9wC2Vxxj9ezyAA7g,2719
pyodbc-5.0.0b3.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
